servers/cromwell/jobs/controllers/utils/task_statuses.py

```diff
@@ -0,0 +1,52 @@
+from jobs.common import enum
+
+# The execution statuses of Cromwell are defined here:
+# https://github.com/broadinstitute/cromwell/blob/710c1931a6745a187ffd026f2cdafea2ffaaf2dc/core/src/main/scala/cromwell/core/ExecutionStatus.scala#L5
+# It provided a reasonable mapping in this file, but may need to be changed in the future on demand
+
+ApiStatus = enum(
+    SUBMITTED='Submitted',
+    RUNNING='Running',
+    ABORTING='Aborting',
+    ABORTED='Aborted',
+    SUCCEEDED='Succeeded',
+    FAILED='Failed')
+CromwellExecutionStatus = enum(
+    NOTSTARTED='NotStarted',
+    QUEUEDINCROMWELL='QueuedInCromwell',
+    STARTING='Starting',
+    RUNNING='Running',
+    ABORTING='Aborting',
+    UNSTARTABLE='Unstartable',
+    ABORTED='Aborted',
+    BYPASSED='Bypassed',
+    RETRYABLEFAILURE='RetryableFailure',
+    FAILED='Failed',
+    DONE='Done')
+
+CROMWELL_EXECUTION_STATUS_MAP = {
+    CromwellExecutionStatus.NOTSTARTED: ApiStatus.SUBMITTED,
+    CromwellExecutionStatus.QUEUEDINCROMWELL: ApiStatus.SUBMITTED,
+    CromwellExecutionStatus.STARTING: ApiStatus.SUBMITTED,
+    CromwellExecutionStatus.RUNNING: ApiStatus.RUNNING,
+    CromwellExecutionStatus.ABORTING: ApiStatus.ABORTING,
+    CromwellExecutionStatus.UNSTARTABLE: ApiStatus.FAILED,
+    CromwellExecutionStatus.ABORTED: ApiStatus.ABORTED,
+    CromwellExecutionStatus.BYPASSED: ApiStatus.SUBMITTED,
```

Comment:
When does a task have a `bypassed` status?

Reply:
Actually, I'm not sure. @jsotobroad and I had some offline experiments and discussion but failed to reproduce this execution status. It seems this status is too rare to appear. Also waiting for Cromwell team's response.